src/js/form.js

```diff
@@ -92,10 +92,13 @@ export function createForm() {
 
   const appendToForm = appendTo(form)
 
+
+
   const formFirstPart = formData
     .flat(1)
     .filter((field) => field.key !== 'reason')
     .filter((field) => !field.isHidden)
+    .filter((field) => field.group == 1)
```

Comment:
here we should sort using the field group. and use a modulo 2 to separate each row...